src/python_framework_v02/troute/nhd_io.py

```diff
@@ -393,13 +393,28 @@ def preprocess_time_station_index(xd):
     unique_times_str = np.unique(xd.time.values).tolist()
 
     unique_times = np.array(unique_times_str, dtype="str")
+    
+    center_time = xd.sliceCenterTimeUTC
```

Comment:
grabbing the sliceCenterTime information from each TimeSlice file. Useful for properly concatenating/trimming the usgs_df